### ISSUE TITLE
Support for Host Records?

### ISSUE THREAD

**Issue by Plork** [murrahjm/Infoblox-Classy#3]
There is no support at the moment for host records

New-IBHostRecord / Get-IBHostRecord / Set-IBHostRecord?

**Comment by murrahjm** [comment]
no, haven't gotten around to that one yet, but definitely on the list of to-dos.

**Comment by Plork** [comment]
Well I only did classes once and I am an ops guy from origin not a dev guy, but will give it a go if i get around to it ;)

**Comment by murrahjm** [comment]
heh, me too man, this dev stuff is all new to me.  The trick with host records as I understand it is they contain other records as child objects, so like the A and cname and all that.  Hopefully with all those classes already defined it shouldn't be too difficult to reference them all in the host class definition.

I've sort of been doing the feature branch method of organization here, where I'll branch master for whatever new feature I'm adding, then once that's done and manually tested I'll merge it back to master to kick off the appveyor build and publish to the gallery.  Not sure how that works with multiple people working on it, I assume I'd make the branch then you could do pull requests against that?
